test/Microsoft.AspNet.Routing.Tests/Template/TemplateBinderTests.cs

```diff
@@ -14,116 +16,97 @@ public class TemplateBinderTests
     {
         private readonly IInlineConstraintResolver _inlineConstraintResolver = GetInlineConstraintResolver();
 
-        public static IEnumerable<object[]> EmptyAndNullDefaultValues
-        {
-            get
+        public static TheoryData EmptyAndNullDefaultValues =>
+            new TheoryData<string, RouteValueDictionary, RouteValueDictionary, string>
```

Comment:
Gotta go w/ @rynowak on this one. Let's not churn the code to use C# 6, especially when the result is so ugly. Also makes it hard to see if anything actually needed to change here.